CatalogResourceTest.resolveEntities test fails

A test is failing on the current master code:

```
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   CatalogResourceTest.resolveEntities:116 
Expecting:
  <[]>
to contain only:
  <[EntityV1{id=1234567890, type=ModelType{name=test, version=1}, version=1, data={"test":"1234"}, constraints=[GraylogVersionConstraint{type=server-version, version=>=3.0.0-alpha.6+4ea04f5}]}]>
but could not find the following elements:
  <[EntityV1{id=1234567890, type=ModelType{name=test, version=1}, version=1, data={"test":"1234"}, constraints=[GraylogVersionConstraint{type=server-version, version=>=3.0.0-alpha.6+4ea04f5}]}]>
```

Example build on Jenkins: https://jenkins.torch.sh/job/graylog-project-snapshot/1358/console

Looking at the time when this started happening, I guess this is related to #5434.

--- a/graylog2-server/src/test/java/org/graylog2/rest/resources/system/contentpacks/CatalogResourceTest.java
+++ b/graylog2-server/src/test/java/org/graylog2/rest/resources/system/contentpacks/CatalogResourceTest.java
@@ -48,6 +48,8 @@ import java.util.Optional;
 import java.util.Set;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
@@ -107,7 +109,7 @@ public class CatalogResourceTest {
                 .data(new ObjectNode(JsonNodeFactory.instance).put("test", "1234"))
                 .build();
         when(mockEntityFacade.resolveNativeEntity(entityDescriptor)).thenReturn(entityDescriptors);
-        when(mockEntityFacade.exportEntity(entityDescriptor, EntityDescriptorIds.empty())).thenReturn(Optional.of(entity));
+        when(mockEntityFacade.exportEntity(eq(entityDescriptor), any(EntityDescriptorIds.class))).thenReturn(Optional.of(entity));
 
         final CatalogResolveRequest request = CatalogResolveRequest.create(entityDescriptors.nodes());
 

